conductr_cli/bndl_main.py

```diff
@@ -75,6 +75,11 @@ def build_parser():
                         help='Description to use for the generated ConductR component',
                         default='')
 
+    parser.add_argument('--annotation',
+                        nargs=2,
+                        action='append',
+                        dest='annotations')
+
```

Comment:
Okay, this has been addressed. I've updated the manual test in my initial comment to show the new usage, ` docker save testing-signals | bndl --annotation my-new-annotation=hello --no-shazar | tar x`